app/helpers/spree/admin/orders_helper_decorator.rb

```diff
@@ -5,28 +5,38 @@ def order_links(order)
         @order ||= order
         links = []
         links << edit_order_link unless action_name == "edit"
-        if @order.complete?
-          links << resend_confirmation_link
-          if @order.distributor.can_invoice?
-            links << send_invoice_link_with_url
-          else
-            links << send_invoice_link_without_url
-          end
-          links << print_invoice_link
-          if Spree::Config.enable_receipt_printing?
-            links << print_ticket_link
-            links << select_ticket_printer_link
-          end
-        end
-        if @order.ready_to_ship?
-          links << ship_order_link
-        end
+        links << complete_order_links if @order.complete?
+        links << ship_order_link if @order.ready_to_ship?
         links << cancel_order_link if @order.can_cancel?
         links
       end
 
       private
 
+      def complete_order_links
+        complete_order_links = []
```

Comment:
Nice refactoring! :heart: 

I would probably write it like this:
```rb
def complete_order_links
  [resend_confirmation_link] + invoice_links + ticket_links
end
```

Reply:
ah, very nice! I added it in a new commit.